### PR TITLE
[READY] Update Tour

### PR DESCRIPTION
**Pull request by sn3p**
Update the Tour page, I'm open to suggestions.
You can compare with the [old (current) content](http://www.learningspaces.io/tour/).

* Updated **Your own Learning Community**
* Updated **Collaborate with your Team**
* Added **Challenge your team**
* Added **Ask your Team**
* Added **Everywhere and Anytime**
* Updated **Excellent and Instant Support**

tour learningspaces

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -148,12 +149,40 @@ en:
...
+          LearningSpaces works seamlessly across all of your devices.
+          Learn and share everywhere and at any time.
+          At work, at home or on the go.
+        image: tour/mobile.jpg
+
       - title: Excellent and Instant Support
-        text_md: "Have a question? Our team is here to help. Our average responsetime is around 1 min."
+        text_md: |
+          Have a question? We are here to help.<br>
+          Our average response time is about 1 minute.
```
Comment (fatboypunk) [inline locales/en.yml]: -about

File: locales/en.yml [hunk 2]
```diff
@@ -148,12 +149,40 @@ en:
...
+        text_md: |
+          Every team member has their own profile.
+          Check it out to see what they're doing and what they've achieved.
         image: tour/profile.jpg
 
+      - title: Ask your Team
+        text_md: |
+          LearningSpaces is about sharing what you know,
+          but learning often starts with what you don't know.
+          Ask your question and see what others are asking.
```
Comment (fatboypunk) [inline locales/en.yml]: > see what others are asking.

Where you can help others?

**Comment by fritsypeij** [comment]
At **Spaces**: "At home, at work or on the road"
At **Everywhere and anytime**: "At work, at home or on the go"

**Comment by fatboypunk** [comment]
> At Spaces: "At home, at work or on the road"
> At Everywhere and anytime: "At work, at home or on the go"

you can comment on the code :)

**Comment by sn3p** [comment]
Anne suggests the following for Spaces:

> Spaces are the key to your team’s knowledge library. Create your own courses and guides or embed existing material. Share, collaborate and discuss with your team! Together you can create and continuously evolve your own community of knowledge.

**Comment by RonaldDijkstra** [comment]
Nice

**Comment by sn3p** [comment]
Ready 🎉 

Result after a small tweak in Anne her suggestion:

screen shot 2016-08-04 at 09 47 22

**Pull request merged**